src/Persistence.LinnApps/Repositories/PartRepository.cs

```diff
@@ -42,7 +42,7 @@ public Part FindById(int key)
 
         public IQueryable<Part> FindAll()
         {
-            return this.serviceDbContext.Parts;
+            return this.serviceDbContext.Parts.AsNoTracking();
```

Comment:
What does asNoTracking do again?

Reply:
doesn't cache anything so you can't update them but takes up less memory  and time

Reply:
we should maybe/probably do this by default for getalls